src/hotspot/cpu/aarch64/nativeInst_aarch64.cpp

```diff
@@ -372,6 +374,7 @@ void NativeCallTrampolineStub::set_destination(address new_destination) {
 void NativeCall::trampoline_jump(CodeBuffer &cbuf, address dest, JVMCI_TRAPS) {
   MacroAssembler a(&cbuf);
 
+  MACOS_AARCH64_ONLY(os::thread_wx_enable_write());
```

Comment:
Harmless, but already handled by set_destination().  I'm not sure about emit_trampoline_stub().

Reply:
I'm not sure about emit_trampoline_stub() either, but I'll leave that one.